Applications/Spire/Source/Ui/ListItem.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Spire/Ui/ListItem.hpp"
 #include <QEvent>
+#include <QFocusEvent>
```

Comment:
When including one header file ```A``` that is required to include header file ```B```, remove all includes of ```B``` to avoid implicit duplicate and redundant includes.